NEWS

```diff
@@ -11,6 +11,9 @@ CHANGES WITH 235:
           that case users will be prevented from correctly managing bond0
           interface using networkd.
 
+        * clock file is now moved to /var/lib/systemd/timesync/clock.
+          Distributors should change the location of the file if it is packaged.
+
```

Comment:
Thank you for the suggestion. I will update the NEWS entry based on that.
> I don't think this file is packaged with systemd.

For example, http://pkgs.fedoraproject.org/rpms/systemd/blob/master/f/systemd.spec#_688